angular/index.d.ts

```diff
@@ -756,7 +756,7 @@ declare namespace angular {
          * @param comparator Function used to determine the relative order of value pairs.
          * @return An array containing the items from the specified collection, ordered by a comparator function based on the values computed using the expression predicate.
          */
-        <T>(array: T[], expression: string|((value: T) => any)|(((value: T) => any)|string)[], reverse?: boolean, comparator?: IFilterOrderByComparatorFunc): T[];
```

Comment:
It returns `null` or `undefined` if it's passed `null` or `undefined`.
To see it, go to [this page](https://docs.angularjs.org/api/), open the DevTools console and run:
```js
angular.injector(['ng']).get('orderByFilter')([1,7,5,3]); // [1,3,5,7]
angular.injector(['ng']).get('orderByFilter')(null); // null
```